contrib/tcsh/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "tcsh"
+pkgver = "6.24.13"
+pkgrel = 0
+build_style = "configure"
+hostmakedepends = ["autoconf", "automake"]
+makedepends = ["ncurses-devel"]
+pkgdesc = "Enhanced version of the Berkeley UNIX C shell"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "BSD-2-Clause"
+url = "https://astron.com/pub/tcsh"
+source = f"{url}/tcsh-{pkgver}.tar.gz"
+sha256 = "1e927d52e9c85d162bf985f24d13c6ccede9beb880d86fec492ed15480a5c71a"
+# check: requires both the tcsh binary and locales to be installed in /usr/* (read only at check step)
+options = ["!check"]
+
+
+def install(self):
+    self.do("make", "-C", "nls", f"prefix={self.chroot_destdir}/usr", "install")
```

Comment:
it has a normal `make install` so i'm not sure why you're calling this manually

prefix=destdir/usr is also incorrect

you can make this post_install and remove this .do and it becomes correct since the makefile already does this, and installs the things below too, so all you actuall yneed to install is the Copyright and install_shell

Reply:
i.e.

```
def post_install(self):
    self.install_license("Copyright")
    self.install_shell("/usr/bin/tcsh")
```

Reply:
> it has a normal `make install` so i'm not sure why you're calling this manually
> 
> prefix=destdir/usr is also incorrect
> 
> you can make this post_install and remove this .do and it becomes correct since the makefile already does this, and installs the things below too, so all you actuall yneed to install is the Copyright and install_shell

For some reason it just refused to work in my environment, I really could not get the thing installed without that